packages/sit-onyx/src/components/OnyxAppLayout/OnyxAppLayout.vue

```diff
@@ -33,55 +33,57 @@ const slots = defineSlots<{
 </template>
 
 <style lang="scss">
+@use "../../styles/mixins/layers.scss";
```

Comment:
```suggestion
@use "@/styles/mixins/layers.scss";
```

(optional) Its maybe easier to import it like so :) Same for all the other components then

Reply:
Done